earth_enterprise/src/installer/common.sh

```diff
@@ -250,6 +250,10 @@ create_links()
         ln -s $BASEINSTALLDIR_VAR/run $BASEINSTALLDIR_OPT/run
     fi
 
+    if [ ! -L "$BASEINSTALLDIR_OPT/share/doc" ]; then
```

Comment:
yeah. that's the thing I was talking about this am. the wrong check I left in there . sending one that will fix it.